README.md

```diff
@@ -86,25 +92,23 @@ First, install and run one of the [supported consensus clients](#consensus-clien
 
 #### Kiln testnet
 
-Here are general setup instructions for Kiln: https://notes.ethereum.org/@launchpad/kiln
```

Comment:
Why is the link to the setup instructions for Kiln removed?

Reply:
it's not the right place, the setup instructions are in other places